examples/jupyter/Quickstart.ipynb

```diff
@@ -431,8 +517,8 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "minuitif = pwa.MinuitIF()\n",
-    "result = minuitif.optimize(estimator, initial_parameters)"
+    "optimizers = pwa.MinuitIF()\n",
```

Comment:
I think `minuit_optimizer` or at least `optimizer` is better

Reply:
Yup weird typo, fixed